moveit_core/collision_detection/src/world.cpp

```diff
@@ -136,6 +136,51 @@ bool World::hasObject(const std::string& id) const
   return objects_.find(id) != objects_.end();
 }
 
+bool World::knowsTransform(const std::string& id) const
+{
+  // Check object names first
+  if (objects_.find(id) != objects_.end())
+    return true;
+  else  // Then objects' named frames
```

Comment:
no need for else here

Reply:
True, but I tend to use explicit else blocks because I find the intent clearer. Is there a style guideline where this is decided?

Some more words on this: https://softwareengineering.stackexchange.com/questions/255523/do-else-blocks-increase-code-complexity

Reply:
Any input from others on this?